controllers/storage.go

```diff
@@ -61,10 +61,13 @@ type Storage struct {
 	// ArtifactRetentionRecords is the maximum number of artifacts to be kept in
 	// storage after a garbage collection.
 	ArtifactRetentionRecords int `json:"artifactRetentionRecords"`
+
+	// ArtifactMaxSize sets the max size in bytes for an artifact contents.
+	ArtifactMaxSize int64 `json:"artifactMaxSize"`
```

Comment:
Maybe it'd he helpful for the reader/user of the API if this field's description mentions about disabling the limit.